scripts/generator/graphql_generator/csharp/SDK/Android/AndroidWebCheckout.cs.erb

```diff
@@ -0,0 +1,41 @@
+namespace <%= namespace %>.SDK.Android {
+    #if UNITY_ANDROID
```

Comment:
I put `#if UNITY_ANDROID` below the namespace because this is the only file defining the namespace